tests/Rector/Contrib/PHPUnit/SpecificMethodCountRector/Correct/correct.php.inc

```diff
@@ -5,5 +5,12 @@ final class MyTest extends \PHPUnit\Framework\TestCase
     public function test()
     {
         $this->assertCount(5, $something);
```

Comment:
@TomasVotruba Gonna add a 3º argument, so we can test the custom error message isn't overwrite.